ext/ftp/ftp.c

```diff
@@ -1387,6 +1387,22 @@ ftp_getresp(ftpbuf_t *ftp)
 }
 /* }}} */
 
+static ssize_t my_send_wrapper_with_restart(php_socket_t fd, const void *buf, size_t size, int flags) {
```

Comment:
Maybe?
```suggestion
static ssize_t php_ftp_send_wrapper_with_restart(php_socket_t fd, const void *buf, size_t size, int flags) {
```
And similar for the other ones

Reply:
Given that these are static functions, I don't think a `php_ftp_` prefix is necessary.

Reply:
Indeed what cmb says,  and the fact that the other static functions don't have an php_ftp prefix either